code/LeftAndMain.php

```diff
@@ -429,12 +430,16 @@ public function schema(HTTPRequest $request): HTTPResponse
     /**
      * Creates a successful json response
      */
-    protected function jsonSuccess(int $statusCode, array $data = []): HTTPResponse
+    protected function jsonSuccess(int $statusCode, ?array $data = null): HTTPResponse
```

Comment:
Why not just use empty array as empty value?

Reply:
Sometimes you actually want to return an empty JSON array

Reply:
Discussed in person - the reason reason is so that you can have an empty body, which seems sensible to allow.